### PR TITLE
Add workspaces default editor and component

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Add workspaces default editor and component.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/21353
fixes https://github.com/eclipse/che/issues/21502

### Is it tested? How?

1. Deploy Eclipse Che with this image: ```quay.io/eclipse/che-dashboard:pr-567```
2. Add defaults(patch checluster/eclipse-che). For example:
```
  devEnvironments:
    defaultComponents:
      - container:
          image: 'quay.io/devfile/universal-developer-image:ubi8-latest'
          sourceMapping: /projects
        name: universal-developer-image
    defaultEditor: eclipse/theia/next
```
3. Create a workspace using a GitHub repository without a devfile. 
For example: ```https://github.com/eclipse-che/che-jwtproxy```
4. You should be able to see all default components propagated to the workspace

Знімок екрана 2022-06-24 о 20 27 15

You can test it with https://eclipse-che.apps.cluster-5s2x5.5s2x5.sandbox995.opentlc.com

### CODE REVIEW

File: packages/dashboard-frontend/src/store/Workspaces/cheWorkspaces/index.ts [hunk 1]
```diff
@@ -273,34 +273,14 @@ export const actionCreators: ActionCreators = {
       }
     },
 
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (akurinnoy) [inline packages/dashboard-frontend/src/store/Workspaces/cheWorkspaces/index.ts:276]: Please add also below this method:

```typescript
  /* eslint-enable @typescript-eslint/no-unused-vars */
```

**Review by ibuziuk — APPROVED** [review]
great job :+1:

File: packages/dashboard-frontend/src/store/Workspaces/cheWorkspaces/index.ts (hunk 1 above)
Comment (olexii4) [reply packages/dashboard-frontend/src/store/Workspaces/cheWorkspaces/index.ts:276]: I have fixed it

**Review by akurinnoy — APPROVED** [review]
LGTM

**Pull request merged**